### PR TITLE
feat: file2.txt added

### PR DESCRIPTION
**Pull request by soheilnikroo**
1. I added file2 in text format
2. bitch

### CODE REVIEW

**Review by soheilnikroo — COMMENTED** [review]
ready

**Comment by soheilnikroo** [comment]
;fkfkkf

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+checking pr
```
Comment (soheilnikroo) [inline file2.txt:1]: dadash inja ready

Comment (straight-forward) [reply file2.txt:1]: skdjdfkjsdh

**Pull request merged**